src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString(){
+        return "Address: " + this.value;
```

Comment:
`Address:` is a magic value that you should put it as a static final variable at the top of your program so that modifications can be done easily.